src/main/kotlin/org/rust/lang/core/psi/ext/RsMacroCall.kt

```diff
@@ -113,8 +113,7 @@ private val RsExpr.value: String? get() {
 fun RsMacroCall.findIncludingFile(): RsFile? {
     if (macroName != "include") return null
     val path = includeMacroArgument?.expr?.value ?: return null
-    // TODO: it doesn't work if `include!()` macro call comes from other macro
-    val file = containingFile?.originalFile?.virtualFile ?: return null
+    val file = (expandedFrom ?: this).containingFile?.originalFile?.virtualFile ?: return null
```

Comment:
```suggestion
    val file = (expandedFromRecursively ?: this).containingFile?.originalFile?.virtualFile ?: return null
```

And need to add the corresponding tests